examples/nerdctl-build-context/README.md

```diff
@@ -0,0 +1,13 @@
+# Complex builds demo
```

Comment:
Is this different from Docker?
If same, this should be rather added to Docker's docs?

Reply:
You're right, there's nothing different here. I'll remove the commit